iree_tests/download_remote_files.py

```diff
@@ -103,6 +103,9 @@ def download_azure_remote_file(test_dir: Path, remote_file: str):
                 "- local MD5 hash matches"
             )
             os.symlink(local_file_path, test_dir / remote_file_name)
+            logger.info(
+                f"  Created symlink for '{local_file_path}' to '{test_dir / remote_file_name}' "
+            )
```

Comment:
nit: can drop trailing ` ` (space). I included those in other sections since the string continued on the next line

Reply:
Ah I see. I thought it was some sort of styling format for logging, thanks